collector/src/main/java/org/apache/hertzbeat/collector/collect/ftp/FtpCollectImpl.java

```diff
@@ -41,12 +43,13 @@ public class FtpCollectImpl extends AbstractCollect {
     private static final String ANONYMOUS = "anonymous";
     private static final String PASSWORD = "password";
 
+    @Setter
+    @VisibleForTesting
+    private FTPClient ftpClient = new FTPClient();
```

Comment:
hi FTPClient does not seem to be thread-safe. Writing this way will cause multiple thread collection exceptions.

Reply:
You are right, I have changed the test way.

Thanks sf🤣.
https://stackoverflow.com/questions/72389685/mockito-mockconstruction-does-not-return-the-mocked-object